titan/params/haart.yml

```diff
@@ -0,0 +1,9 @@
+haart:
+  use_cap:
+    default: false
+    description: Whether "prob" defined in demographics.race.sex_type.haart is a percent of diagnosed, HIV+ agents instead of a per-time-step probability for HAART coverage
+    type: boolean
```

Comment:
also add haart aids scalar here, and add method to haart that aids can call to get it